app/renderer/js/main.ts

```diff
@@ -426,6 +426,11 @@ class ServerManagerView {
 		return this.tabs[this.activeTabIndex].webview.props.url;
 	}
 
+	shouldCurrentActiveServerNotify(): boolean {
+		const { notify } = DomainUtil.getDomain(this.activeTabIndex);
+		return notify === undefined ? false : notify;
```

Comment:
This looks confusing. Can you make it a bit clearer (add a comment or two/make the code better).